pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/creator/IndexCreationContext.java

```diff
@@ -393,25 +436,19 @@ public H3IndexConfig getH3IndexConfig() {
   }
 
   class Range extends Wrapper {
-
-    private final Comparable<?> _min;
-    private final Comparable<?> _max;
     private final int _rangeIndexVersion;
 
-
-    Range(IndexCreationContext delegate, int rangeIndexVersion, Comparable<?> min, Comparable<?> max) {
+    Range(IndexCreationContext delegate, int rangeIndexVersion) {
       super(delegate);
       _rangeIndexVersion = rangeIndexVersion;
-      _min = min;
-      _max = max;
     }
 
     public Comparable<?> getMin() {
-      return _min;
+      return getMinValue();
```

Comment:
This is an evolving interface (added recently, never tried to replace a range index yet) so you could just change the method name here i.e. just remove it.